### PR TITLE
Commitactivitybug

### PR DESCRIPTION
**Pull request by josecordaz**

### CODE REVIEW

File: pkg/git/commit.go [hunk 1]
```diff
@@ -22,17 +22,19 @@ type commitActivity struct {
...
+	RenamedFrom       string              `json:"renamed_from"`
+	RenamedTo         string              `json:"renamed_to"`
+	Additions         int                 `json:"additions"`
+	Deletions         int                 `json:"deletions"`
+	Binary            bool                `json:"binary"`
+	Excluded          bool                `json:"excluded"`
+	ExcludedReason    string              `json:"excluded_reason"`
+	License           string              `json:"license"`
+	LicenseConfidence float32             `json:"license_confidence"`
 }
```
Comment (jhaynie) [inline pkg/git/commit.go]: we still need all the blame information. this is meant to be a record of changes (blame) for each file in the commit. so it's missing author_id, sloc, loc, etc.

File: pkg/git/commit.go [hunk 2]
```diff
@@ -22,19 +22,27 @@ type commitActivity struct {
...
+	License            string              `json:"license"`
+	LicenseConfidence  float32             `json:"license_confidence"`
+	Size               int64               `json:"size"`
+	Loc                int64               `json:"loc"`
+	Sloc               int64               `json:"sloc"`
+	Comments           int64               `json:"comments"`
+	Blanks             int64               `json:"blanks"`
+	Complexity         int64               `json:"complexity"`
+	WeightedComplexity float64             `json:"weighted_complexity"`
+	Skipped            string              `json:"skipped"`
```
Comment (jhaynie) [inline pkg/git/commit.go]: shouldn't skipped be a bool?

Comment (jhaynie) [reply pkg/git/commit.go]: let's remove weightedcomplexity

Comment (josecordaz) [reply pkg/git/commit.go]: skipped contains the reason, that is how ripsrc is handled it. https://github.com/pinpt/ripsrc/blob/55e8299e35618e0ff58f8a5a26a91f7e4ae9e488/ripsrc/code_info.go#L74

**Pull request merged**